go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/edgexfoundry/device-snmp-go
 
 require (
-	github.com/edgexfoundry/device-sdk-go v1.4.0
-	github.com/edgexfoundry/go-mod-core-contracts v0.1.115
+	github.com/edgexfoundry/device-sdk-go/v2 v2.0.0-dev.54
```

Comment:
Please wait till recent rework PR has been merged:
https://github.com/edgexfoundry/device-sdk-go/pull/922

Reply:
@hahattan edgexfoundry/device-sdk-go#922 has been merged. Please proceed with this PR using SDK `dev.58` or higher.